File: AICodeDetector/datasets/all/human/linear2.php
<?php 
function linearSearch($arr, $target, $occurrences = false) { 
	$length = count($arr); 
	$indices = []; 

	for ($i = 0; $i < $length; $i++) { 
		if ($arr[$i] == $target) { 
			$indices[] = $i; 

			if (!$countOccurrences) { 
				return $indices; 
			} 
		} 
	} 

	return $occurrences ? count($indices) : $indices; 
} 

// Driver code 
$arr = [10, 20, 30, 40, 50]; 
$targetValue = 30; 

$result = linearSearch($arr, $targetValue); 

if (is_array($result)) { 
	echo "Element found at index "
		. implode(', ', $result); 
} else { 
	echo "Element not found in the array"; 
} 

?>

File: AICodeDetector/datasets/go/train2/human/linear2.go
package main

import "fmt"

func linearSearch(list []int, target int) int {
	for i, num := range list {
		if num == target {
			return i
		}
	}
	return -1
}

func main() {
	searchList := []int{13, 25, 14, 86, 5, 24, 7, 5, 42, 14}
	target := 78

	if index := linearSearch(searchList, target); index != -1 {
		fmt.Printf("Element %d found at index %d\n", target, index)
	} else {
		fmt.Printf("Element %d not found in the list\n", target)
	}
}
